Sources/SwiftSyntax/Syntax.swift

```diff
@@ -464,6 +479,20 @@ extension Syntax {
     return base.tokens
   }
 
+  /// Sequence of `SyntaxClassifiedRange`s for this syntax node.
+  public var classifications: SyntaxClassifications {
+    return base.classifications
+  }
+
+  /// Sequence of `SyntaxClassifiedRange`s contained in this syntax node within
+  /// a relative range.
+  /// - Parameters:
+  ///   - in: The relative byte range to pull `SyntaxClassifiedRange`s from.
+  /// - Returns: Sequence of `SyntaxClassifiedRange`s.
+  public func classifications(in range: ByteSourceRange) -> SyntaxClassifications {
```

Comment:
Could we migrate the location structures here to `SourceRange`?

Reply:
It's easy to get a `ByteSourceRange` from `SourceRange`, the other way is not so convenient.
I would prefer not to require a `SourceRange` in this API but just add a `var byteRange: ByteSourceRange` property to `SourceRange`. If you have a `SourceRange` just pass its `byteRange` property to this API.